03-altalanos.md

```diff
@@ -85,7 +85,7 @@
     3. Az után-akció (találat utáni akció) olyan dupla találat jellegű helyzet, ahol a hibázó vívó a találat kapás pillanatában, vagy azután kezd meg egy közbetámadást, vagy az összetett támadás befejező mozzanatát. Az után-akciókért nem jár pont, és nem teszik semmissé a kapott találatot. 
 
 4.  Találatokat a következő módokon lehet kivitelezni:
-    1. Ütés - A fegyver élével vagy lapjával végrehajtott támadás, ami sérülést okozna az adott területen (például a lappal fejet ért ütés is érvényes).
+    1. Ütés - A fegyver élével vagy lapjával végrehajtott támadás, ami sérülést okozna az adott területen (a megfelelő technikai kivitelezéssel végrehajott lappal bevitt találatok is érvényesek).
```

Comment:
Ez kell ide? merthogy fentebb ott van, hogy ütés == 'sérülést kell okoznia' .

ez a technikai kivitelezés egy elég szőrős dolog szerintem.

Reply:
Az is benne van pár helyen, hogy a bíró hatásköre eldönteni dolgokat, szerintem ezek lefedik a dolgot, nem kell külön hangsúlyozni meg szőrösködni, ilyenek miatt szedtem ki azt is h hány másodpercig lehessen birkózni, mert hacsak nem kezd el a bíró számolni h 1 elefánt 2 elefánt stb, akkor az nem lesz igazán betartható :D

Reply:
Agyaltam rajta én is, hogy kell-e ide.
Hagyjuk ki, szerintem gond úgy sem lesz belőle, hogy reklamálnának a lappal való ütések miatt.

Reply:
Hát aki akadékoskodik, annak megmutatjuk h a bíró szemgolyó használatával ELDÖNTI, hogy érvényesnek ítél-e egy találatot, aminek estét már megállapították. Ha ragaszkodnak hozzá h csalnak, majd megnézzük az asszóját és azután küldjük el a faszba. Vagy ha kell iróniát, sőt, végső esetbe szarkazmust alkalmazunk (ezt a szabályba is foglalhatjuk :P ), és mutatunk nekik modern kard találatokat, a dominancia jegyáben :D

Reply:
igazából ott van az első szavak között a lap is, annyi sztem pont elég